hpfennig.new.rst

```diff
@@ -0,0 +1,165 @@
+------------------------------------------------------------
+summary
+------------------------------------------------------------
+
+.. class:: myname
+
+Herbert Pfennig
+
+.. class:: contact
+
+**E:** dasein555@gmail.com
+**L:** Redwood City, CA
+
+.. class:: summary
+
+Internet handyman with over 14 years of experience working in startups. I
+enjoy solving problems and building infrastructure to help engineers get
+stuff done through automation. Over the years I have built up experience in
+Test Automation, Systems and Network Administration, Internal Tools
+Development and Release Engineering.
+
+------------------------------------------------------------
+skillset
+------------------------------------------------------------
+
+| **OS:**             Linux, OSX, FreeBSD
+| **Platforms:**      Docker, AWS, GCE, Kubernetes, Virtualbox, VMWare
+| **Languages:**      Python, Shell scripting, with some GO, C, C++ etc.
+| **Internet:**       HTTP, SMTP, DNS, DHCP, GPXE, TCP/IP, UDP
+| **Tools:**          Git, Gerrit, Github Enterprise, Jenkins, GNUmake, BSDMake,
+                      Ansible, Terraform, Packer, Sumologic, Datadog, Quay
+| **Learning:**       Concourse CI, GO
+
+------------
+employment
+------------
+
+.. class:: jobtitle
+..
+
+	*[07/2014 - Current]* :: **Release\Tools Engineer** :: `Bracket Computing`_
```

Comment:
Depending on what positions you want do apply for next - you could consider putting something like 'Release Tools and Infrastructure Engineer'[1]  here maybe. You did much more than just ReleaseTools!

[1] Take all English advices from me with a grain of salt - I'm not a native speaker and I'm not sure if this is an actual title! 😄

Reply:
I prefixed it with build, since that was my official title....although I did more, not sure I can make up my own titles ;)